### ISSUE TITLE
A11y documentation audit: Review Back-to-top component documentation

### ISSUE THREAD

**Issue by k80bowman** [department-of-veterans-affairs/va.gov-team#35996]
## Description
We want to audit our accessibility documentation on design.va.gov to make sure that it is up-to-date with current accessibility guidelines.

For this ticket, we need to review the [accessibility guidelines for the Back-to-top component](https://design.va.gov/components/back-to-top#accessibility-considerations) on design.va.gov and provide any updates that need to be made to the existing documentation as a comment on this ticket.

## Acceptance Criteria
- [ ] Review accessibility guidelines on the following pages of design.va.gov:
    - [ ] https://design.va.gov/components/back-to-top#accessibility-considerations
- [ ] Add notes on any necessary updates as a comment on this ticket
- [ ] Ping Katy, Carol, and Natalie for review

**Comment by GnatalieH** [comment]
Just want to note the Back to top has a couple of known a11y issues in that it lives outside the DOM and doesn't have a specific-enough text label (not great for screenreader users), and it is a link styled as a button (not materially honest).

**Comment by CourtneyChambers77** [comment]
I added a few more considerations for this element 
Animations can be distracting to users with vestibular conditions.
Some users may have smooth scrolling turned off in their preferences, which will affect the transition going back to the top of the page.
After the user has clicked Back to top, the code should call focus on an element near the top of the page, such as the H1.
When using the Back to Top button for pages ensure they are longer than 4 screens.
For relatively short page lengths, Back to Top links are not needed
Only one sticky is needed, Back to Top link per page instead of including multiple links in every section of the page. 
Note: This position is out of the way, but noticeable.
